core/shared/src/main/scala/scalaz/zio/Async.scala

```diff
@@ -25,5 +25,5 @@ object Async extends Serializable {
   /**
    * Constructs an `Async` that represents a synchronous return.
    */
-  final def now[E, A](io: IO[E, A]): Async[E, A] = Now(io)
+  final def succeed[E, A](io: IO[E, A]): Async[E, A] = Now(io)
```

Comment:
This one should be left as `now`, because it indicates an immediate return value (_now_), contrasted with a "later" return value.

Reply:
Likewise, `succeedLazy` becomes `nowLazy`?

Reply:
ha, I didn't see it was `Async`, not `IO`.

Reply:
:heavy_check_mark: